src/main/java/dev/jbang/util/WinJunction.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright 2011 杨博 (Yang Bo)
+ * Translated from Scala to Java 2024 (Tako Schotanus)
```

Comment:
where is this from?

Reply:
https://github.com/Atry/scala-junction/blob/master/src/main/scala/com/dongxiguo/junction/Junction.scala